composeml/label_times.py

```diff
@@ -191,13 +191,21 @@ def bin(self, bins, quantiles=False, labels=None, right=True):
             cutoff_time           2014-01-01 00:45:00  2014-01-01 00:48:00
             my_labeling_function                 high                  low
         """ # noqa
-        data = self.copy()
-        name = data.settings['name']
+        label_times = self.copy()
+        name = label_times.settings['name']
+        values = label_times[name].values
 
         if quantiles:
-            data[name] = pd.qcut(data[name].values, q=bins, labels=labels)
+            label_times[name] = pd.qcut(values, q=bins, labels=labels)
 
         else:
-            data[name] = pd.cut(data[name].values, bins=bins, labels=labels, right=right)
+            label_times[name] = pd.cut(values, bins=bins, labels=labels, right=right)
 
-        return data
+        label_times.settings.update({
```

Comment:
perhaps these settings should be nested. under a single key, since they refer to a single transform

also, do we need to update the describe to properly display these new settings nicely?

Reply:
nevermind, looks like you do this in https://github.com/FeatureLabs/compose-ml/pull/27